### PR TITLE
adds a linting of all shell files the project

### PR DESCRIPTION
**Pull request by GregMeno**
Signed-off-by: Gregory Meno <gmeno@redhat.com>

### CODE REVIEW

File: ceph-docker-lint/config/definitions/ceph-docker-lint.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            - gmeno
+          white-list:
+            - xarses
+            - angdraug
+            - mbroz
+            - ddiss
+            - osynge
+          org-list:
+            - ceph
+          trigger-phrase: ''
```
Comment (andrewschoen) [inline ceph-docker-lint/config/definitions/ceph-docker-lint.yml]: Could you add a trigger phrase here?

File: ceph-docker-lint/config/definitions/ceph-docker-lint.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      artifactDaysToKeep: -1
+      artifactNumToKeep: -1
+
+    parameters:
+      - string:
+          name: sha1
+          description: "A pull request ID, like 'origin/pr/72/head'"
+
+    triggers:
+      - github-pull-request:
```
Comment (andrewschoen) [inline ceph-docker-lint/config/definitions/ceph-docker-lint.yml]: status lines would be helpful to differentiate this from the other pr tests. like https://github.com/ceph/ceph-build/blob/master/ceph-docker-prs/config/definitions/ceph-docker-prs.yml#L46-L49

File: ceph-docker-lint/config/definitions/ceph-docker-lint.yml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+          auto-close-on-fail: false
+
+    scm:
+      - ceph-docker
+
+    builders:
+      - shell:
+          !include-raw:
+            - ../../../scripts/build_utils.sh
+            - ../../build/setup
```
Comment (andrewschoen) [inline ceph-docker-lint/config/definitions/ceph-docker-lint.yml]: I don't see a ``../../build/setup`` file here.

File: ceph-docker-lint/config/definitions/ceph-docker-lint.yml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+          permit-all: false
+          auto-close-on-fail: false
+
+    scm:
+      - ceph-docker
+
+    builders:
+      - shell:
+          !include-raw:
+            - ../../../scripts/build_utils.sh
```
Comment (andrewschoen) [inline ceph-docker-lint/config/definitions/ceph-docker-lint.yml]: This doesn't hurt, but it doesn't look like you're using anything here from that file.

File: ceph-docker-lint/config/definitions/ceph-docker-lint.yml (hunk 2 above)
Comment (GregMeno) [inline ceph-docker-lint/config/definitions/ceph-docker-lint.yml]: done

File: ceph-docker-lint/config/definitions/ceph-docker-lint.yml (hunk 1 above)
Comment (GregMeno) [inline ceph-docker-lint/config/definitions/ceph-docker-lint.yml]: done

File: ceph-docker-lint/config/definitions/ceph-docker-lint.yml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+            - gmeno
+          white-list:
+            - xarses
+            - angdraug
+            - mbroz
+            - ddiss
+            - osynge
+          org-list:
+            - ceph
+          trigger-phrase: 'jeninks lint'
```
Comment (andrewschoen) [inline ceph-docker-lint/config/definitions/ceph-docker-lint.yml]: s/jeninks/jenkins

Comment (GregMeno) [inline ceph-docker-lint/config/definitions/ceph-docker-lint.yml]: hah, thank you

**Pull request merged**